packaging/macOS/macos_pyinstaller.sh

```diff
@@ -0,0 +1,25 @@
+# Note : pyinstaller doesn't work with python 3.8 yet.
+# see https://github.com/pyinstaller/pyinstaller/issues/4311
+
+cd ../..
```

Comment:
cd to relative directory is not a good idea given you don't know from where this script is going to be run:
```sh
$ pwd
/home/menahem
$ ./parsec-cloud/packaging/macOS/macos_pyinstaller.sh
!!! crash because cd ../.. leads to / given we are running the script from /home/menahem/
```

The solution is usually to do 
```
BASEDIR=$(dirname $(readlink -f "$0") )
cd $BASEDIR/../..
```

Besides, instead of using cd, you should check pushd/popd ;-)